### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by mih**

### CODE REVIEW

File: docs/basics/101-116-sharelocal.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ how a dataset can be obtained from a path (instead of a URL as shown in the sect
 :ref:`installds`). Thirdly, ``DataLad-101`` is a dataset that can
 showcase many different properties of a dataset already, but it will
 be an additional learning experience to see how the different parts
-of the dataset -- text files, larger files, datalad subdataset,
+of the dataset -- text files, larger files, :dlcmd:`subdataset`,
```
Comment (adswa) [inline docs/basics/101-116-sharelocal.rst:55]: Shouldn't this say "subdatasets" and not be a command?

File: docs/basics/101-180-FAQ.rst [hunk 2]
```diff
@@ -369,7 +369,7 @@ If you do not want to invent a description yourself, you can run
 
       Get the dataset
 
-      A DataLad dataset can be "cloned" by running 'datalad clone <url>'.
+      A DataLad dataset can be "cloned" by running ``datalad clone <url>``.
```
Comment (adswa) [inline docs/basics/101-180-FAQ.rst:372]: I'm unsure whether the backticks are really needed in this format of the template

File: docs/basics/101-116-sharelocal.rst (hunk 1 above)
Comment (mih) [reply docs/basics/101-116-sharelocal.rst:55]: The sentence is complicated. This was my interpretation.

File: docs/basics/101-180-FAQ.rst (hunk 2 above)
Comment (mih) [reply docs/basics/101-180-FAQ.rst:372]: ok, will revert

File: docs/basics/101-116-sharelocal.rst (hunk 1 above)
Comment (mih) [reply docs/basics/101-116-sharelocal.rst:55]: Updated

**Review by adswa — APPROVED** [review]
thanks!

**Pull request merged**